jdaviz/configs/cubeviz/plugins/cube_listener.py

```diff
@@ -63,14 +67,24 @@ def sonify_spectrum(spec, duration, overlap=0.05, system='mono', srate=44100, fm
 
 class CubeListenerData:
     def __init__(self, cube, wlens, samplerate=44100, duration=1, overlap=0.05, buffsize=1024,
-                 bdepth=16, wl_unit=None, audfrqmin=50, audfrqmax=1500, eln=False, vol=None):
+                 bdepth=16, wl_unit=None, audfrqmin=50, audfrqmax=1500, eln=False, vol=None,
+                 spectral_axis_index=2, n_cpu=None):
         self.siglen = int(samplerate*(duration-overlap))
         self.cube = cube
         self.dur = duration
         self.bdepth = bdepth
         self.srate = samplerate
         self.maxval = pow(2, bdepth-1) - 1
         self.fadedx = 0
+        if n_cpu is None:
+            self.n_cpu = mp.cpu_count() - 1
```

Comment:
is it ever possible that this would result in `n_cpu = 0` and we should worry about that?  Is subtracting the one to leave one CPU for the manager/kernel?